bin/git-info

```diff
@@ -1,11 +1,26 @@
 #!/usr/bin/env bash
 
 get_config() {
-  git config --list
+  cmd_get_config="$(git config --get-all extras.info.config-grep)"
```

Comment:
Now we prefer to use `git-extras` as the namespace, like `git config --get git-extras.pr.remote`.

Reply:
Fixed